test/source/tests/decrypt.ts

```diff
@@ -800,6 +800,17 @@ XZ8r4OC6sguP/yozWlkG+7dDxsgKQVBENeG6Lw==
       await BrowserRecipe.pgpBlockVerifyDecryptedContent(t, browser, { params, content: [], encryption: '', signature: '', error: 'parse error' });
     }));
 
+    ava.default('decrypt - prevent rendering of attachments from domain sources other than flowcrypt.s3.amazonaws.com1', testWithBrowser('compatibility', async (t, browser) => {
+      const threadId1 = '184cc6aa8e884397';
+      const acctEmail = 'flowcrypt.compatibility@gmail.com';
+      const inboxPage = await browser.newPage(t, TestUrls.extension(`chrome/settings/inbox/inbox.htm?acctEmail=${acctEmail}&threadId=${threadId1}`));
+      await inboxPage.waitAll('iframe');
+      const pgpBlock = await inboxPage.getFrame(['pgp_block.htm']);
+      await pgpBlock.waitForSelTestState('ready');
+      await pgpBlock.waitForContent('@pgp-block-content', '[skipped attachment due to invalid url]');
+      await pgpBlock.notPresent(['.preview-attachment', '@download-attachment-0']);
```

Comment:
Good, you added a positive test -- the warning should appear.
I added a negative test -- the attachment should not appear.